index.html

```diff
@@ -270,26 +276,26 @@ <h3>Multi-Post Stories</h3>
 
         <!--Contact Forms-->
         <div class="contact flame">
-          <div class="Mname"><h1>CONTACT US</h1></div>
+          <div class="Cname"><h1>CONTACT US</h1></div>
 
           <div class="WKdetail flame">
             <p>
               If you have an Application you are interested in developing, a
-              Feature that you need built or a Project that need Coding. I would
-              Love to help with it..
+              Feature that you need built or a Project that need Coding,    I 'd
+              Love to help with it.
             </p>
           </div>
           <div class="frmC">
-            <form action="submit" id="contactF" class="contact">
-              <input type="text" id="names" placeholder="Text Input" /> <br />
-              <input type="text" id="email" placeholder="Jeremy@gmail.com" />
+            <form action="https://formspree.io/f/mwkydljv" method="post" id="contactF" class="contact">
+              <input type="text" id="names" name="names" placeholder="Text Input" required/> <br />
```

Comment:
- [ ] According to the requirements, the name field should accept up to 30 chars, so kindly add a maxlength attribute for this input element. `maxlength="30"`

![1](https://user-images.githubusercontent.com/45231276/177534734-5ca47ff2-82e4-4e71-9c96-f716e7565aa3.PNG)